examples/resources/gateway_route_expression.tf

```diff
@@ -0,0 +1,10 @@
+resource "konnect_gateway_route_expression" "expression_route" {
+  name       = "AnythingExpression"
```

Comment:
Since there are more properties that users can specify (even if nullable) - would it help if the examples contain all of them?